code/modules/reagents/reagents/food-Drinks.dm

```diff
@@ -360,6 +360,10 @@
 	scannable = TRUE
 	common = TRUE
 
+/datum/reagent/sodiumchloride/affect_blood(mob/living/carbon/M, alien, effect_multiplier) //reasoning: Table salt is usually iodized. Iodine saturates glands. Glands are often getting screwed over by isotopes of iodine if exposed to radiation. Yes its a preventatitve method IRL but this is gamyfictaion.
```

Comment:
Why is this in blood?
isnt the point of iodine to collect rads then expel them later?
Blood implies that the blood is using rad-collected salt

Reply:
I was just thinking having it on ingest would fit better ;-;